### DISCUSSION TITLE
模组中关于异想体事件的选择时罪人对不同概率的回应似乎不是很完善？

### DISCUSSION THREAD

**Discussion by silence61** [LocalizeLimbusCompany/LocalizeLimbusCompany#193] [问答 | Q&A]
在翻WIKI的时候发现其实罪人对于异想体事件判定时不同的概率会有不同的文本，但模组内对概率高和极高与低和极低时罪人的回应只有一种文本，如以实玛丽在极高概率的情况下文本为“应该可行。如果你的计划不会像艘破烂搁浅的船一样。”但实际上文本为“问我行不行？别特地问些理所应当的事。”（来自边狱巴士的WIKI），希望零协能够尽快完善这方面问题，感谢全体汉化组人员的付出。

**Comment by N071** [comment]
首先，灰机Wiki的文本填入正是参考了我们模组中 [AbDlg_Ishmael.json](https://paratranz.cn/projects/6860/strings?file=888985) 文件的译文内容。
我们在翻译 AbDlg_Ishmael.json 时，确实见到并翻译了您提到的“应该可行。如果你的计划不会像艘破烂搁浅的船一样。”这句和其他更多句子。我们在翻译方面应该不存在文本丢失的情况。

与此同时，我们的汉化程序并没有(也无法)触动边狱公司本身的代码，因此安装模组应当是不会导致回应文本变化的。
目前看来，这个问题**大概率**并非我们所致。

综上所述，这个问题应当还需要更加稳定的复现，尤其是在非汉化原版条件的复现，来确认问题的发生情况与发生位置。我们可以自己尝试或者联系灰机Wiki方面帮忙。

**Reply by silence61** [reply]
了解，所以看来又是月记的祖传代码问题了。

**Reply by N071** [reply]
值得一提的是，判定概率只分5类，分别是：绝对优势、优势、均势、劣势、绝对劣势。
而人格的判定回应条目数一般都在5种以上。以我们刚刚提到的 AbDlg_Ishmael.json 为例，其内的判定回应文本共有7句。额外的句子有可能仅在特定的关卡或遭遇中生效。

**Reply by silence61** [reply]
OK，感谢零协的耐心解答。
